python/ray/ml/checkpoint.py

```diff
@@ -0,0 +1,112 @@
+import abc
+
+import ray
+
+
+class Checkpoint(abc.ABC):
+    """Checkpoint interface"""
```

Comment:
We should expand on these docs later; I'm assuming they are bare-bones for expediency.

Reply:
Yes!